src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+import { PeoplePersonHeader } from '../../types/PeopleTableHeader';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const { personSlug = '' } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {Object.entries(PeoplePersonHeader).map(([key, value]) => (
```

Comment:
we won't reuse `PeoplePersonHeader` , so it's better to just create array with the headers instead of enum